src/unix/kqueue.c

```diff
@@ -34,6 +34,10 @@
 #include <fcntl.h>
 #include <time.h>
 
+#ifndef EV_OOBAND
+#define EV_OOBAND  EV_FLAG1
```

Comment:
Ah, I guess I misunderstood your "The issue is Mac OS X" comment.  Never mind then.